index.html

```diff
@@ -28,6 +28,11 @@ <h1>Dev Delta Jokes</h1>
         <li><a href="sam-mcknight/index.html">Sam's Joke Page</a></li>
         <li><a href="sean/joke.html">Sean's Joke Page</a></li> 
         <li>Shannon Placeholder</li>
+<<<<<<< HEAD
+        <li><a href="./Jonah/index.html">Jonah's Joke Page</a></li>
+=======
+        <li><a href="Jonah/index.html">Jonah's Joke Page</a></li>
+>>>>>>> ad786ff48bb63743ea8882e5637a9f29108bd425
```

Comment:
Great work on the refactor Jonah! What you're seeing here though is an example of a merge conflict, and it's telling you that this line of code changed in two different commits, and it doesn't know which one to keep and which one to discard, so you have to manually make that change to decide for it. 

```suggestion
        <li><a href="Jonah/index.html">Jonah's Joke Page</a></li>
```